src/app/view-component/group-views/group-info/group-manage/group-manage.component.ts

```diff
@@ -104,4 +116,29 @@ export class GroupManageComponent implements OnInit {
     
   }
 
+  onSort(event: SortEvent) {
+    console.log(event);
```

Comment:
I suppose that was for debugging, you should clean your code afterwards.

Reply:
Okay

Reply:
Removed the comments